inst/templates/get-started.Rmd

```diff
@@ -383,6 +383,49 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculate coverage statistics
+
+```{r}
+# Matched companies based on suggested matches
+matched_rows <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow()
+
+# Share of matched companies from total loanbook companies
+matched_share <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow() / loanbook %>%
+    distinct(company_name) %>%
+    nrow()
+matched_share
+
+# Number and share of matched companies classified by misc_info from total loanbook companies
+# misc_info should be replaced with variables like sectors, headcount etc.
+misc_share <- loanbook %>%
+  count(misc_info) %>%
+  left_join(to_edit %>%
+    filter(suggest_match) %>%
+    count(misc_info), by = c("misc_info"), suffix = c("_loanbook", "_merged")) %>%
+  mutate(n_share = n_merged / n_loanbook)
+
+# Calculate loan exposure categorised based on misc_info
+# id_tilt column is used as the substitute due to unavailability of loan amount column in sample data. Please replace id_tilt column with the loan amount column.
```

Comment:
Please wrap your lines to fit in 80 characters. In RStudio you can do that automatically with "shift + ctrl/cmd + /".